packages/docusaurus-utils/src/cliUtils.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import prompts from 'prompts';
+import logger from '@docusaurus/logger';
+
+export type LanguagesOptions = {
+  javascript?: boolean;
+  typescript?: boolean;
+};
+
+export async function getLanguage(
```

Comment:
This function is only used in create-docusaurus. Please don't put it in utils.

Reply:
Hey, I did this extraction because of @slorber commentary telling that this will be reused in another part of the code, if it is true, I can maybe pass the prompt function as a parameter and remove the dependency, what do you think about it? I can also bring it back to create-docusaurus!

Reply:
yes, considering we also want this for swizzle it's fine to add it to docusaurus utils

see also related pr: https://github.com/facebook/docusaurus/pull/9681